CHANGELOG.md

```diff
@@ -0,0 +1,40 @@
+# 0.3.0
+
+This version roughly corresponds with all the changes made to the Temporal polyfill as of the November TC39 Plenary meeting.
+
+Backwards-incompatible changes:
+ - Timezones now require a `getOffsetNanosecondsFor` method, and no longer fall back to the intrinsic definition (previously provided by `Temporal.Timezone#getOffsetNanosecondsFor`) if not provided. 08346dc5
```

Comment:
I think commit IDs in MD files need to be HTTPS links, otherwise they'll show up as plain text to the reader.

Reply:
BTW I'm not sure about this; in just not seeing the commits show up as links when editing here on my phone.

Reply:
Unfortunately you are right:  https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls notes that this doesn't happen for wikis or files within repos.

Reply:
I've added all the commit links, and added some notes in the release-checklist on how to quickly regenerate the list of links.